Loop/Managers/DeviceDataManager.swift

```diff
@@ -536,8 +536,14 @@ extension DeviceDataManager {
         return pumpManager?.status
     }
 
-    var sensorState: SensorDisplayable? {
-        return cgmManager?.sensorState
+    func sensorState(for glucose: GlucoseSampleValue?) -> SensorDisplayable? {
+        if let glucose = glucose, glucose.wasUserEntered {
+            // the CGM manager needs to determine the glucoseValueType for a manual glucose based on its managed glucose thresholds
+            let glucoseValueType = (cgmManager as? CGMManagerUI)?.glucoseValueType(for: glucose)
```

Comment:
It's probably not right that we're asking the CGM to categorize the manual glucose value. It suggest that maybe glucose categorization should be a Loop concern.  What do we do if we don't have a CGM?  If this is too big, let's not tackle it here, but I think this is a sign we need to refactor glucose range categorization.

Reply:
@ps2 The scenario when the CGM is not available was reported to design. They agree that ideally Loop manages the user glucose thresholds. However, they don't have a big concern with the style of the manually entered glucose being `normal` in the scenario when no CGM is setup. However, they would like to confirm there is no risk in a build at some point.